themes-default/slim/views/apiBuilder.mako

```diff
@@ -4,156 +4,149 @@
 %>
 <!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8">
-    <meta name="robots" content="noindex, nofollow">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width">
-    <!-- These values come from css/dark.css and css/light.css -->
-    % if sbThemeName == "dark":
-    <meta name="theme-color" content="#333333">
-    % elif sbThemeName == "light":
-    <meta name="theme-color" content="#333333">
-    % endif
-    <title>Medusa - ${title}</title>
-    <base href="${base_url}">
-    <!--[if lt IE 9]>
-    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
-    <meta name="msapplication-TileColor" content="#FFFFFF">
-    <meta name="msapplication-TileImage" content="images/ico/favicon-144.png">
-    <meta name="msapplication-config" content="css/browserconfig.xml">
-    <link rel="shortcut icon" href="images/ico/favicon.ico">
-    <link rel="icon" sizes="16x16 32x32 64x64" href="images/ico/favicon.ico">
-    <link rel="icon" type="image/png" sizes="196x196" href="images/ico/favicon-196.png">
-    <link rel="icon" type="image/png" sizes="160x160" href="images/ico/favicon-160.png">
-    <link rel="icon" type="image/png" sizes="96x96" href="images/ico/favicon-96.png">
-    <link rel="icon" type="image/png" sizes="64x64" href="images/ico/favicon-64.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="images/ico/favicon-32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="images/ico/favicon-16.png">
-    <link rel="apple-touch-icon" sizes="152x152" href="images/ico/favicon-152.png">
-    <link rel="apple-touch-icon" sizes="144x144" href="images/ico/favicon-144.png">
-    <link rel="apple-touch-icon" sizes="120x120" href="images/ico/favicon-120.png">
-    <link rel="apple-touch-icon" sizes="114x114" href="images/ico/favicon-114.png">
-    <link rel="apple-touch-icon" sizes="76x76" href="images/ico/favicon-76.png">
-    <link rel="apple-touch-icon" sizes="72x72" href="images/ico/favicon-72.png">
-    <link rel="apple-touch-icon" href="images/ico/favicon-57.png">
+    <head>
+        <meta charset="utf-8">
+        <meta name="robots" content="noindex, nofollow">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <!-- These values come from css/dark.css and css/light.css -->
+        % if app.THEME_NAME == "dark":
```

Comment:
Since these use the same content is there much point in keeping this?

Reply:
Good point.
I don't think that meta var is even used, so I'll remove those from both files.
Plus `dark.css` / `light.css` are not really a thing anymore (they are now `themed.css` for each theme respectively)

Reply:
I wonder if there are other parts of the CSS we could now look at removing since the themes are "separate".